packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -33,8 +33,22 @@ const MIN_GAS_PRICE = 3; // Safelow gas price from GasStation, in Gwei
 @withEthBalance // ETH balance
 @observer
 class Send extends Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      gas: 0
+    };
+    this.isUnmounted = false;
+  }
+
+  componentWillUnmount () {
+    this.isUnmounted = true;
```

Comment:
Kind of ugly, but unfortunately necessary :(. Can't find any other simple solutions. @axelchalon any ideas?

Reply:
React officially recommends cancellable promises: https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html. But we can keep this way actually, there's less code.